conf/config.yaml

```diff
@@ -56,6 +56,11 @@ apisix:
   # port_admin: 9180              # use a separate port
   # https_admin: true             # enable HTTPS when use a separate port for Admin API.
                                   # Admin API will use conf/apisix_admin_api.crt and conf/apisix_admin_api.key as certificate.
+  mtls:
+    mtls_enable: false          # Enable or disable mTLS. Enable depends on `port_admin` and `https_admin`.
```

Comment:
I don't think we need this switch, if user set cert and key, then enable it.

Reply:
We could set `mtls_enable ` false  to disable this feature, and retain the example config.
If remove this switch, we have to remove the example config.
What do you think ?

Reply:
we can set empty string for disable them as default.